README.md

```diff
@@ -36,7 +40,10 @@ Below is a car engine audio clip, and two two versions of the same clip, each on
 - [car-engine__eq2](/assets/car-engine__eq2.wav?raw=1)
 
 #### Your Answer
-<!-- Write your answer here -->
+<!--
+EQ1) Lower pitch
+EQ2) Higher pitch
+ -->
```

Comment:
So, these clips are both the same _fundemental_ pitch a the original, but EQ1 has had higher frequencies removed, and EQ2 had the lower frequencies removed. We'll go over EQ in class tonight.